deepcell/model_zoo/fpn.py

```diff
@@ -132,9 +132,18 @@ def create_pyramid_level(backbone_input,
                 [pyramid, upsamplelike_input])
         else:
             upsampling = UpSampling2D if ndim == 2 else UpSampling3D
-            size = (2, 2) if ndim == 2 else (1, 2, 2)
-            pyramid_upsample = upsampling(size=size, name=upsample_name,
-                                          interpolation=interpolation)(pyramid)
+            if ndim == 2:
+                upsampling_kwargs = {
+                    'size': (2, 2),
+                    'name': upsample_name,
+                    'interpolation': interpolation
+                }
+            else:
+                upsampling_kwargs = {
+                    'size': (1, 2, 2),
+                    'name': upsample_name
+                }
+            pyramid_upsample = upsampling(**upsampling_kwargs)(pyramid)
```

Comment:
I think it might be cleaner to leave the old `size` logic and just remove interpolation if ndim != 3.

```python
size = (2, 2) if ndim == 2 else (1, 2, 2)
upsampling_kwargs = {
    'size': size,
    'name': upsample_name,
    'interpolation': interpolation
}
if ndim > 2:
    del upsampling_kwargs['interpolation']
```